src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateResponseSealedSpec.java

```diff
@@ -90,20 +90,19 @@ private CallbackResponse prepareSealedForm(CallbackParams callbackParams) {
                 caseData
             );
             assignCategoryId.assignCategoryIdToCaseDocument(stitchedDocument, DocCategory.DEF1_DEFENSE_DQ.getValue());
-            CaseDocument stitchedDocumentCopy = assignCategoryId.copyCaseDocumentWithCategoryId(
-                    stitchedDocument, DocCategory.DQ_DEF1.getValue());
+            CaseDocument stitchedDocumentCopy = assignCategoryId.copyCaseDocumentWithCategoryId(stitchedDocument, DocCategory.DQ_DEF1.getValue());
             if (nonNull(caseData.getRespondent2DocumentGeneration()) && caseData.getRespondent2DocumentGeneration().equals("userRespondent2")) {
```

Comment:
("userRespondent2").equals(caseData.getRespondent2DocumentGeneration())
replace this 2 check in 1 = equals check null then return false